Added ActionButton.

millevlada opened the pull request:
Added disabled reason for PopupMenu2 options, so it can better reuse IUIAction. Also added options to use text and hint as plain text. Fixed z-index for popup menu given inside floating panels.

millevlada committed:
Added ActionButton.
Added disabled reason for PopupMenu2 options, so it can better reuse IUIAction. Also added options to use text and hint as plain text.
Fixed z-index for popup menu given inside floating panels.

fjalvingh commented on to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java:
```diff
@@ -0,0 +1,68 @@
+		HR hr = new HR();
+		hr.addCssClass("act-rule");
+		add(hr);
+		NodeBase actionButton = Icon.faEllipsisV.createNode();
```
If the button is disabled the menu needs to be disabled too.

fjalvingh replied:
On a button of this type the icon is not a ... but a v (down triangle)

millevlada replied:
ok, fixed

fjalvingh commented on to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java:
```diff
@@ -0,0 +1,68 @@
+		m_instance = instance;
+	}
+
+	public ActionButton<T> addAction(IUIAction<T> action) {
```
Why would all instances need to be a "T"? I think both the primary action and the additional actions could have different instances?

millevlada replied:
fixed

millevlada committed:
Disabling of additional actions when button is disabled.
Changed popup icon to shevron down.
All options can have their own objects for actions.
